frontend/packages/dev-console/integration-tests/support/constants/pageTitle.ts

```diff
@@ -33,6 +33,7 @@ export const pageTitle = {
   BuilderImages: 'Builder Images',
   HelmCharts: 'Helm Charts',
   InstallHelmCharts: 'Install Helm Chart',
+  CreateHelmRelease: 'Create Helm Release',
```

Comment:
Do we need InstallHelmCharts as we are shifting to CreateHelmRelease?

Reply:
No we do not, have went in and removed it.